docs/getting_started/tutorial.md

```diff
@@ -0,0 +1,867 @@
+# Your first Wagtail site
+
+```eval_rst
+.. note::
+   This tutorial covers setting up a brand new Wagtail project.
+   If you'd like to add Wagtail to an existing Django project instead, see :doc:`integrating_into_django`.
+```
+
+## Install and run Wagtail
+
+### Install dependencies
+
+Wagtail supports Python 3.6, 3.7, 3.8 and 3.9.
+
+To check whether you have an appropriate version of Python 3:
+
+```sh
+$ python3 --version
+```
+
+If this does not return a version number or returns a version lower than 3.6, you will need to [install Python 3](https://www.python.org/downloads/).
+
+```eval_rst
+.. important::
+   Before installing Wagtail, it is necessary to install the **libjpeg** and **zlib** libraries, which provide support for working with JPEG, PNG and GIF images (via the Python **Pillow** library).
+   The way to do this varies by platform—see Pillow's
+   `platform-specific installation instructions <https://pillow.readthedocs.org/en/latest/installation.html#external-libraries>`_.
+```
+
+### Create and activate a virtual environment
+
+We recommend using a virtual environment, which provides an isolated Python environment.
+This tutorial uses [venv](https://docs.python.org/3/tutorial/venv.html), which is packaged with Python 3.
+
+**On Windows** (cmd.exe):
+
+```bat
+$ python3 -m venv mysite\env
+$ mysite\env\Scripts\activate.bat
+```
+
+**On Unix or MacOS** (bash):
+
+```sh
+$ python3 -m venv mysite/env
+$ source mysite/env/bin/activate
+```
+
+**For other shells** see the [venv documentation](https://docs.python.org/3/library/venv.html).
```

Comment:
venv, should be surrounded by backticks

Reply:
Added in 0c4d82ac3c99e688c02e51d321cf8b4669e6c486